src/main/resources/static/schema.graphql

```diff
@@ -115,6 +117,14 @@ type Item {
     lastAccessedDate: DateTime
     categoryId: Int
 }
+input ItemDTO {
```

Comment:
Please change all fields here except "imagePath" to required fields to be consistent with corresponding REST API.